community_supplied/Kernel/kernel-tcpip/check-arp-cache-mgmt-count.rule

```diff
@@ -0,0 +1,180 @@
+/*
+ * This rule checks for the neighbour ARP cache count on management interface.
+ *
+ * It shows yellow alarm when the count exceeds 90 percent of the maximum count.
+ * It shows red alarm when the count exceeds 95 percent of the maximum count or if there is an increase in drop count.
+ * It's sampling time is 1 minute.
+ */
+iceberg {
+    topic kernel.tcpip {
```

Comment:
Please take care of all above review comments in below rules

Reply:
Modified icerberg to healthbot